react/Menu/index.jsx

```diff
@@ -1,9 +1,10 @@
 import React, { Component } from 'react'
+
 import styles from './styles.styl'
 import cx from 'classnames'
 import PropTypes from 'prop-types'
 import { Media, Bd, Img } from '../Media'
-
+import PopOver from '../PopOver'
```

Comment:
Popover without capital O please, otherwise we will always make a mistake, and those on macOS will not see the mistake, and will see it in Travis.